src/error.rs

```diff
@@ -75,6 +79,9 @@ pub enum SlateDBError {
 
     #[error("Read channel error: {0}")]
     ReadChannelError(#[from] tokio::sync::oneshot::error::RecvError),
+
+    #[error("Merge Operator is not fully implemented")]
+    MergeUnsupported,
```

Comment:
shall we just panic in cases where we'd raise this instead of adding an error to the api?

Reply:
fixed